### ISSUE TITLE
[Bug]: vsprintf(): Too few arguments in L10NString.php:84

### ISSUE THREAD

**Issue by tgoeg** [nextcloud/server#31166]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Continuation of #20562

I also applied the patch in the mentioned issue.

This is the detailed output that the increased debug info generates (shouldn't this be part of the default distribution so error messages have the needed details right from the start?):
```
{"reqId":"yeKYUF1nSCk53XldsKh6","level":3,"time":"2022-02-14T11:54:28+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"vsprintf(): Too few arguments at /var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php#84","userAgent":"--","version":"22.2.3.0","exception":{"Exception":"Error","Message":"vsprintf(): Too few arguments at /var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php#84","Code":0,"Trace":[{"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php","line":84,"function":"vsprintf"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/L10N.php","line":106,"function":"__toString","class":"OC\\L10N\\L10NString","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/LazyL10N.php","line":52,"function":"t","class":"OC\\L10N\\L10N","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":537,"function":"t","class":"OC\\L10N\\LazyL10N","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":397,"function":"sendNoSubscriptionEmail","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":345,"function":"handleNoSubscription","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/BackgroundJobs/CheckSubscription.php","line":47,"function":"checkSubscription","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/BackgroundJob/Job.php","line":51,"function":"run","class":"OCA\\Support\\BackgroundJobs\\CheckSubscription","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/BackgroundJob/TimedJob.php","line":58,"function":"execute","class":"OC\\BackgroundJob\\Job","type":"->"},{"file":"/var/www/nextcloud.my.domain/cron.php","line":127,"function":"execute","class":"OC\\BackgroundJob\\TimedJob","type":"->"}],"File":"/var/www/nextcloud.my.domain/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"yeKYUF1nSCk53XldsKh6","level":3,"time":"2022-02-14T11:54:28+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Unable to format \"Ein Nextcloud-Abonnement f\u00fcr Organisationen bedeutet, dass sich die Nextcloud-Entwickler zu 100 % dem Erfolg Ihres Cloud-Servers widmen: der Sicherheit, Skalierbarkeit, Leistung und Funktionalit\u00e4t Ihres eigenen Dienstes!\" with \"\"","userAgent":"--","version":"22.2.3.0"}
{"reqId":"yeKYUF1nSCk53XldsKh6","level":3,"time":"2022-02-14T11:54:28+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"vsprintf(): Too few arguments at /var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php#84","userAgent":"--","version":"22.2.3.0","exception":{"Exception":"Error","Message":"vsprintf(): Too few arguments at /var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php#84","Code":0,"Trace":[{"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/L10NString.php","line":84,"function":"vsprintf"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/L10N.php","line":106,"function":"__toString","class":"OC\\L10N\\L10NString","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/L10N/LazyL10N.php","line":52,"function":"t","class":"OC\\L10N\\L10N","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":537,"function":"t","class":"OC\\L10N\\LazyL10N","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":397,"function":"sendNoSubscriptionEmail","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/Service/SubscriptionService.php","line":345,"function":"handleNoSubscription","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/apps/support/lib/BackgroundJobs/CheckSubscription.php","line":47,"function":"checkSubscription","class":"OCA\\Support\\Service\\SubscriptionService","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/BackgroundJob/Job.php","line":51,"function":"run","class":"OCA\\Support\\BackgroundJobs\\CheckSubscription","type":"->"},{"file":"/var/www/nextcloud.my.domain/lib/private/BackgroundJob/TimedJob.php","line":58,"function":"execute","class":"OC\\BackgroundJob\\Job","type":"->"},{"file":"/var/www/nextcloud.my.domain/cron.php","line":127,"function":"execute","class":"OC\\BackgroundJob\\TimedJob","type":"->"}],"File":"/var/www/nextcloud.my.domain/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"yeKYUF1nSCk53XldsKh6","level":3,"time":"2022-02-14T11:54:28+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Unable to format \"Ein Nextcloud-Unternehmens-Abonnement bedeutet, dass sich die Entwickler hinter Deinem privaten Cloud-Server zu 100 % Deinem Erfolg widmen: der Sicherheit, Skalierbarkeit, Leistung und Funktionalit\u00e4t Deines Dienstes!\" with \"\"","userAgent":"--","version":"22.2.3.0"}
```

### Steps to reproduce

1. Have an Admin with German locale (I guess?)
2. Update NC 21.x to 22.2.3.0
3. Run cron
4. Get mentioned error in the log

### Expected behavior

Translation should work, cron should run successfully. I'll link another ticket I have yet to create. This has broader effects on cron.

### Installation method

Manual installation

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 7.4

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "config_is_read_only": "true",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "my.nextcloud.domain"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/my.nextcloud.domain",
        "dbtype": "mysql",
        "version": "22.2.3.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": false,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0
        },
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "log_rotate_size": 10000000,
        "loglevel": 3,
        "logfile": "\/home\/nextcloud_user\/nextcloud.log",
        "logfilemode": 432,
        "log.condition": {
            "apps": [
                "admin_audit"
            ]
        },
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "maintenance": false,
        "updater.release.channel": "stable",
        "trashbin_retention_obligation": "auto, 14",
        "mail_sendmailmode": "smtp",
        "app_install_overwrite": [
            "impersonate",
            "spreed",
            "files_accesscontrol",
            "quota_warning",
            "richdocuments"
        ],
        "encryption.key_storage_migrated": false,
        "defaultapp": "files"
    }
}
```

### List of activated Apps

```shell
Enabled:
  - accessibility: 1.8.0
  - activity: 2.15.0
  - admin_audit: 1.12.0
  - bruteforcesettings: 2.2.0
  - calendar: 3.0.1
  - circles: 22.1.1
  - cloud_federation_api: 1.5.0
  - comments: 1.12.0
  - contactsinteraction: 1.3.0
  - dav: 1.19.0
  - encryption: 2.10.0
  - federatedfilesharing: 1.12.0
  - federation: 1.12.0
  - files: 1.17.0
  - files_accesscontrol: 1.11.1
  - files_downloadactivity: 1.12.0
  - files_pdfviewer: 2.3.1
  - files_rightclick: 1.1.0
  - files_sharing: 1.14.0
  - files_trashbin: 1.12.0
  - files_versions: 1.15.0
  - files_videoplayer: 1.11.0
  - firstrunwizard: 2.11.0
  - groupquota: 0.1.7
  - impersonate: 1.8.0
  - logreader: 2.7.0
  - lookup_server_connector: 1.10.0
  - nextcloud_announcements: 1.11.0
  - notifications: 2.10.1
  - oauth2: 1.10.0
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.12.0
  - quota_warning: 1.13.0
  - recommendations: 1.1.0
  - serverinfo: 1.12.0
  - settings: true
  - sharebymail: 1.12.0
  - support: 1.5.0
  - survey_client: 1.10.0
  - systemtags: 1.12.0
  - text: 3.3.0
  - theming: 1.13.0
  - theming_customcss: 1.10.0
  - twofactor_backupcodes: 1.11.0
  - updatenotification: 1.12.0
  - user_ldap: 1.12.1
  - viewer: 1.6.0
  - workflowengine: 2.4.0
Disabled:
  - dashboard
  - files_external
  - files_trackdownloads
  - spreed
  - user_status
  - weather_status
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
see above
```

**Comment by Kokokokoka** [comment]
can confirm this issue, on our server there were many infinitely-working cron tabs with la of 120 on 6 cores:
[core] Error: Error while running background job (class: OCA\Support\BackgroundJobs\CheckSubscription, arguments: )
at 2022-02-15T15:42:15+03:00
 ```
 | ValueError: The arguments array must contain 1 items, 0 given/var/www/nextcloud/lib/private/L10N/L10NString.php - line 82: vsprintf()/var/www/nextcloud/lib/private/L10N/L10N.php - line 106: OC\L10N\L10NString->__toString()/var/www/nextcloud/lib/private/L10N/LazyL10N.php - line 52: OC\L10N\L10N->t()/var/www/nextcloud/apps/support/lib/Service/SubscriptionService.php - line 536: OC\L10N\LazyL10N->t()/var/www/nextcloud/apps/support/lib/Service/SubscriptionService.php - line 396: OCA\Support\Service\SubscriptionService->sendNoSubscriptionEmail()/var/www/nextcloud/apps/support/lib/Service/SubscriptionService.php - line 344: OCA\Support\Service\SubscriptionService->handleNoSubscription()/var/www/nextcloud/apps/support/lib/BackgroundJobs/CheckSubscription.php - line 47: OCA\Support\Service\SubscriptionService->checkSubscription()/var/www/nextcloud/lib/private/BackgroundJob/Job.php - line 51: OCA\Support\BackgroundJobs\CheckSubscription->run()/var/www/nextcloud/lib/private/BackgroundJob/TimedJob.php - line 58: OC\BackgroundJob\Job->execute()/var/www/nextcloud/cron.php - line 128: OC\BackgroundJob\TimedJob->execute()
-- | --
```
AJAX fails too.
part of config:
```
$CONFIG = array (
  'datadirectory' => '/var/nc_data',
  'dbtype' => 'mysql',
  'version' => '23.0.0.10',
  'dbname' => 'nextcloud',
  'dbhost' => 'localhost:/var/run/mysqld/mysqld.sock',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
  'mysql.utf8mb4' => true,
  'installed' => true,
  'activity_expire_days' => 14,
  'auth.bruteforce.protection.enabled' => true,
  'blacklisted_files' =>
  array (
    0 => '.htaccess',
    1 => 'Thumbs.db',
    2 => 'thumbs.db',
  ),
  'cron_log' => true,
  'default_phone_region' => 'RU',
  'enable_previews' => true,
  'enabledPreviewProviders' =>
  array (
    0 => 'OC\\Preview\\PNG',
    1 => 'OC\\Preview\\JPEG',
    2 => 'OC\\Preview\\GIF',
    3 => 'OC\\Preview\\BMP',
    4 => 'OC\\Preview\\XBitmap',
    5 => 'OC\\Preview\\Movie',
    6 => 'OC\\Preview\\PDF',
    7 => 'OC\\Preview\\MP3',
    8 => 'OC\\Preview\\TXT',
    9 => 'OC\\Preview\\MarkDown',
  ),
  'filesystem_check_changes' => 0,
  'filelocking.enabled' => 'true',
  'htaccess.RewriteBase' => '/',
  'integrity.check.disabled' => false,
  'knowledgebaseenabled' => false,
  'logfile' => '/var/nc_data/nextcloud.log',
  'loglevel' => 2,
  'logtimezone' => 'Europe/Moscow',
  'log_rotate_size' => 100 * 1024 * 1024,
  'maintenance' => false,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'memcache.locking' => '\\OC\\Memcache\\Redis',
  'overwriteprotocol' => 'https',
  'preview_max_x' => 1024,
  'preview_max_y' => 768,
  'preview_max_scale_factor' => 1,
  'redis' =>
  array (
    'host' => '/var/run/redis/redis-server.sock',
    'port' => 0,
    'timeout' => 0.0,
  ),
  'quota_include_external_storage' => false,
  'share_folder' => '/Shares',
  'skeletondirectory' => '',
  'theme' => '',
  'trashbin_retention_obligation' => 'auto, 7',
  'updater.release.channel' => 'stable',
  'ldapProviderFactory' => 'OCA\\User_LDAP\\LDAPProviderFactory',
  'app_install_overwrite' =>
  array (
    0 => 'files_trackdownloads',
    1 => 'sharepermissions',
    2 => 'files_mindmap',
    3 => 'bruteforcesettings',
  ),
);


```

**Comment by Kokokokoka** [comment]
can confirm that this issue is fixed in 23.0.2
